File: Daily Test/C++/Maximum Oranges - Equal Count 21.2.2026.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n, m = 0;
    cin >> n;

    vector<int> l(n);
    for(int i = 0; i < n; i++)
        cin >> l[i];

    for(int i = 0; i < n; i++) {
        int d = l[i];
        for(int j = i; j < n; j++)
            d = min(d, l[j]);

        int t = d * (n - i);
        m = max(m, t);
    }

    cout << m;
    return 0;
}

File: Daily Test/Java/Maximum Oranges - Equal Count 21.2.2026.java
import java.util.*;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int[] l = new int[n];

        for(int i = 0; i < n; i++)
            l[i] = sc.nextInt();

        int m = 0;

        for(int i = 0; i < n; i++) {
            int d = l[i];

            for(int j = i; j < n; j++)
                d = Math.min(d, l[j]);

            int t = d * (n - i);
            m = Math.max(m, t);
        }

        System.out.print(m);
        sc.close();
    }
}